### ISSUE TITLE
Bitcoin: Incoming RBF Replacement Transaction Shows as "failed"

### ISSUE THREAD

**Issue by bitschmidty** [breadwallet/breadwallet-ios#87]
If I receive an RBF enabled (original) transaction it shows in the transaction list as expected.

If the original transaction is replaced (replacement transaction), it too shows up in the transaction list. However this replacement transaction has a "failed" label associated with it. In testing, the “Failed” transaction actually ends up being the transaction that confirms.

![rbf-brd-transaction-list-incoming-bumped 1](https://user-images.githubusercontent.com/1615772/51925144-0bc42480-23b4-11e9-9b6c-e8a8f38823c0.png)

**Comment by Izzyhudson** [comment]
Except

**Comment by akshaynexus** [comment]
does it update afterwards without the failed label?ie after the failed tx is confirmed.

**Comment by bitschmidty** [comment]
@akshaynexus Yes, I believe it eventually did. Not sure after how many confirmations.
